pkg/cmd/step/pr/step_pr_comment.go

```diff
@@ -120,5 +122,12 @@ func (o *StepPRCommentOptions) Run() error {
 		Number: &prNumber,
 	}
 
+	if o.Flags.Code {
+		return provider.AddPRComment(&pr, escapeAsCode(o.Flags.Comment))
+	}
 	return provider.AddPRComment(&pr, o.Flags.Comment)
 }
+
+func escapeAsCode(comment string) string {
+	return "```\n" + comment
```

Comment:
what's about the closing backticks?

Reply:
you don't seem to need to close them on github, it works without it.

Reply:
it works without on GitHub, but that does not mean that it is the correct syntax. GitHub is just more lenient. It might not work in other cases. Also, it just works if you treat the whole comment as code.

Or image a tool which tries to download the comments and parse the markdown. It might not be as lenient either.

There is a difference between "it works" and doing the right thing.

Reply:
👎